src/Internal/EventStoreCatchUpSubscription.php

```diff
@@ -55,9 +53,9 @@ abstract class EventStoreCatchUpSubscription implements SyncEventStoreCatchUpSub
     private ?DropData $dropData;
     private bool $allowProcessing;
     private bool $isProcessing;
-    protected bool $shouldStop;
-    private bool $isDropped;
-    private bool $stopped;
+    protected bool $shouldStop = false;
+    private bool $isDropped = false;
+    private bool $stopped = false;
```

Comment:
This property is already initialized in the constructor and also it's initialized with true there. So this doesn't make sense.

Reply:
Indeed, the initial state of the subscription is stopped, not started.

Reply:
When I was running the code php complained about this line as well. I've changed the value and going to push it.